src/promptflow-evals/promptflow/evals/evaluators/_content_safety/common/evaluate_with_rai_service.py

```diff
@@ -40,15 +58,11 @@ async def ensure_service_availability(rai_svc_url: str, token: str, capability:
     :type capability: str
     :raises Exception: If the service is not available in the region or the capability is not available.
     """
-    headers = {
-        "Authorization": f"Bearer {token}",
-        "Content-Type": "application/json",
-        "User-Agent": USER_AGENT,
-    }
-
+    headers = get_common_headers(token)
     svc_liveness_url = rai_svc_url + "/checkannotation"
+
     async with httpx.AsyncClient() as client:
-        response = await client.get(svc_liveness_url, headers=headers)
+        response = await client.get(svc_liveness_url, headers=headers, timeout=60)
```

Comment:
It seems a bit odd to have a hardcoded timeout like this. Should we make a follow-up task to replace all timeouts with a constant and/or environment variable?

Reply:
Sure, will do!